app/main.py

```diff
@@ -2,0 +1,15 @@
+
+def copy_file(command: str) -> None:
+    command_parsed = command.split(" ")
+    file_in_name = command_parsed[1]
+    file_out_name = command_parsed[2]
```

Comment:
The variable name 'file_out_name' could be more descriptive. A better name could be 'destination_file_name'.